diff --git a/setup.py b/setup.py
index c7fa6a858..9682cdc1c 100644
--- a/setup.py
+++ b/setup.py
@@ -31,7 +31,7 @@ from setuptools import find_packages, setup
 from setuptools.command.test import test as TestCommand
 
 install_requires = [
-    'b2handle>=1.1.1,<2.0.0',
+    # 'b2handle>=1.1.1,<2.0.0',
     'elasticsearch<3.0.0,>=2.0.0',
     'elasticsearch-dsl<3.0.0,>=2.0.0',
     'datacite>=0.3.0',
